### ISSUE TITLE
Fan stop

### ISSUE THREAD

**Issue by Kenzyss** [ilya-zlobintsev/LACT#351]
### Checklist

- [X] The problem is not listed in the [hardware support matrix](https://github.com/ilya-zlobintsev/LACT?tab=readme-ov-file#hardware-support) as a known limitation
- [X] I've included a [debug snapshot](https://github.com/ilya-zlobintsev/LACT?tab=readme-ov-file#reporting-issues) if the issue is related to hardware configuration

### Bug description

[LACT-sysfs-snapshot-20240731-181915.tar.gz](https://github.com/user-attachments/files/16444638/LACT-sysfs-snapshot-20240731-181915.tar.gz)
If you lower the fan speed slider to 0, the fans do not stop, but operate at minimum speed. If you enable automatic fan control, they stop (up to a certain temperature). Perhaps this is not a bug, but an unimplemented feature
![image](https://github.com/user-attachments/assets/799e6a28-378a-488e-acac-3adf502f419f)
![image](https://github.com/user-attachments/assets/3429b9b7-406b-462b-8a0e-5bbc874f8cfa)

### System info

```markdown
- LACT version: 0.5.5 release
- GPU model:AMD RADEON RX 6800 XT
- Kernel version: 6.9.12-lqx1-1-lqx
- Distribution: Arch
```

**Comment by ilya-zlobintsev** [comment]
Do they actually not stop physically, or just reported at a speed above 0? It's pretty common that 0 RPM isn't measured correctly by the GPU whem using a custom curve, while actually working fine.

**Comment by Kenzyss** [comment]
> Do they actually not stop physically, or just reported at a speed above 0? It's pretty common that 0 RPM isn't measured correctly by the GPU whem using a custom curve, while actually working fine.

They don't physically stop. In corectrl there is something like fan stop, the statistics say 0 rpm, but in reality the fans are still working

**Comment by ilya-zlobintsev** [comment]
If you set it to 0 RPM, but it doesn't actually stop (and it happens in corectrl as well), then it's likely a GPU firmware issue.

**Comment by StarTroop** [comment]
I have the same issue with my Dell 6600 xt. In corectl the fan stop _does_ work. Only in LACT does the fan not totally spin down (minimum around 31%, 1010 rpm for me). Checking `/sys/class/drm/card1/device/hwmon/hwmon1/fan1_input` and `/sys/class/drm/card1/device/hwmon/hwmon1/fan1_target` whenever I manually set fan speed to anything under 31% always shows a number around 1010, never below.  
I can hear the fan spinning so I know it's not just a reporting bug.

**Comment by Kenzyss** [comment]
Yes, it's probably a hardware limitation. When I set the fans to static (with pwm=0), then
cat /sys/class/drm/card1/device/hwmon/hwmon1/pwm1
cat /sys/class/drm/card1/device/hwmon/hwmon1/pwm1_min
are equal to 0 (as in auto mode), but

cat /sys/class/drm/card1/device/hwmon/hwmon1/fan1_input
>>520
cat /sys/class/drm/card1/device/hwmon/hwmon1/fan1_target
>>516
cat /sys/class/drm/card1/device/hwmon/hwmon1/fan1_min
>>0

That is, fan1_min = 0, but for some reason the fan speed is equal to fan1_target(~520rpm)
In auto mode fan1_target = 0

Maybe there is some way to solve this problem?
AMD GPU RX 6800xt (Reference)
